docs/managing_deis/upgrading-deis.rst

```diff
@@ -63,6 +63,56 @@ Finally, update ``deisctl`` to the new version and reinstall:
     enable PROXY protocol for ports 80 and 443 on the ELB, add a ``TCP 443:443`` listener, and
     change existing targets and health checks from HTTP to TCP.
 
+
+Graceful Upgrade
+----------------
+
+Alternatively, an experimental feature exists to provide the ability to perform a graceful upgrade. Unlike the
+in-place process above, this process keeps the platforms routers and publishers up during the upgrade process.
+This means that there should only be a maximum of around 1-2 seconds of downtime while the routers boot up. Many
```

Comment:
> Ill re-add something short and generic to explain the concern

Got it.  My only concern was that the following sounds a bit non-deterministic, so a brief explanation would help:

> This means that there should only be a maximum of around 1-2 seconds of downtime while the routers boot up. Many times, there will be no downtime at all.